src/main/java/org/sagebionetworks/bridge/util/IntegTestUtils.java

```diff
@@ -22,15 +23,14 @@
     public static final String SAGE_ID = "sage-bionetworks";
     public static final String SAGE_NAME = "Sage Bionetworks";
 
-    public static void deletePhoneUser(TestUserHelper.TestUser researcher) throws IOException {
-        checkArgument(researcher.getRoles().contains(RESEARCHER));
+    public static void deletePhoneUser(TestUserHelper.TestUser admin) throws IOException {
+        checkArgument(admin.getRoles().contains(ADMIN) || admin.getRoles().contains(SUPERADMIN));
```

Comment:
Why pass in the admin when we can just call TestUserHelper.getSignedInAdmin() directly?